README.adoc

```diff
@@ -78,6 +78,50 @@ You can also download it from the https://github.com/pgrange/bash_unit/releases[
 
 endif::[]
 
+== pre-commit hook
+
+You can run `+bash_unit+` as a pre-commit hook.
+
+Add the following to your pre-commit configuration. By default it will run scripts that are identified as shell scripts that match the path `+^tests/(.*/)?test_.*\.sh$+`.
+
+[.pre-commit-config,yaml]
+----
+repos:
+  - repo: https://github.com/pgrange/bash_unit
+    rev: v1.7.0
+    hooks:
+      - id: bash-unit
+----
+
+== Running `+bash_unit+` in the test script
```

Comment:
I would remove this section in the documentation as test_doc, in my opinion, is really advanced and odd use of bash_unit. One should not have to do that in 99.99% of the cases so that describing it here introduces some sort of noise in the doc, in my opinion and can even be a bit scary for new users :)

Reply:
FYI, I added bash_unit in these projects:

- https://github.com/mdeweerd/pre-commit-completion/blob/main/tests/test_pre_commit_completion.sh
- https://github.com/mdeweerd/pre-commit-php/blob/mdw/tests/test_doc.sh

But I may be the 0.00001% ;-).